Mathlib/RingTheory/Adjoin/PowerBasis.lean

```diff
@@ -145,17 +145,16 @@ theorem repr_pow_isIntegral [IsDomain S] (hB : IsIntegral R B.gen) {x : A}
     ∀ i, IsIntegral R (B.basis.repr (x ^ n) i) := by
   nontriviality A using Subsingleton.elim (x ^ n) 0, isIntegral_zero
   revert hx
```

Comment:
since we're now using the induction tactic, it looks like this `revert` and then `intro` become unneeded?